src/cloudant/document.py

```diff
@@ -351,5 +401,5 @@ def put_attachment(self, attachment, content_type, data, headers=None):
             headers=headers
         )
         resp.raise_for_status()
-
+        self.fetch()
```

Comment:
Having seen this second example am I correct in thinking that the second `fetch()` call is necessary to update the doc with the new attachment data?